logback-core/src/test/java/ch/qos/logback/core/net/AbstractSocketAppenderTest.java

```diff
@@ -381,27 +374,42 @@ public void postProcessesEventsBeforeTransformingItToASerializable() throws Exce
     appender.start();
     awaitStartOfEventDispatching();
 
-    // when
-    appender.append("some event");
+    appendEvent();
     awaitAtLeastOneEventToBeDispatched();
 
     // then
     InOrder inOrder = inOrder(appender, preSerializationTransformer);
-    inOrder.verify(appender).postProcessEvent("some event");
-    inOrder.verify(preSerializationTransformer).transform("some event");
+    inOrder.verify(appender).postProcessEvent(event);
+    inOrder.verify(preSerializationTransformer).transform(event);
+  }
+
+  @Test
+  public void prepareEventsForDeferredProcessingBeforeTransferingToDeque() throws Exception {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/tony19/logback/file/2371903498/issues/source?bid=3222618&fileBranchId=3222681#l387)